tmereader/repository.go

```diff
@@ -7,6 +7,23 @@ import (
 	"sync"
 )
 
+type TmeSource int
+
+const (
+	AuthorityFiles TmeSource = iota
+	KnowledgeBases
+)
+
+var sourceName = [...]string{
+	"authorityfiles",
+	"knowledgebases",
+}
+
+var sourcePathSuffix = [...]string{
+	"terms",
+	"eng/categories",
+}
+
```

Comment:
Couldn't we group all the knowledge base related info in one struct (for example), and the authority file related info in another structure? For me this way of defining enums seems a little bit complicated to understand and follow...